<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pipeToolWindow</class>
 <widget class="QWidget" name="pipeToolWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>377</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>377</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>377</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>管线生产工具</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icon/SynBuilder.ico</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="page0" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>导入导出</string>
    </property>
    <widget class="QPushButton" name="choosePipeDataBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>611</width>
       <height>231</height>
      </rect>
     </property>
     <property name="title">
      <string>导入管点模型文件</string>
     </property>
     <widget class="QListWidget" name="modelListWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>501</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteModelBtn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>180</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectAllModelBtn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>130</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>全选/反选</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseFolderModelBtn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打开文件夹</string>
      </property>
     </widget>
     <widget class="QPushButton" name="chooseSingleModelBtn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="chooseSaveBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>213</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>数据集名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="datasetEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>261</x>
       <y>290</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>导出路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="savePathLineEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>511</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>选择管线数据：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="vectorDataEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="prePushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextPushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okPushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>341</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QWidget" name="page1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>641</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>管线颜色配置</string>
    </property>
    <widget class="QTableWidget" name="colorTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>511</width>
       <height>281</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>80</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>选中</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>父类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>父名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>子类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>子名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设置颜色</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addColorBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seleAllColorBtn">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>全选/反选</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteColor">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>170</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QPushButton" name="defaultColor">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>默认</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="page2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>管线字段映射</string>
    </property>
    <widget class="QTableWidget" name="lineTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>281</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <column>
      <property name="text">
       <string>目标表关联字段名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>导入表关联字段名</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="page3" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>641</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>管点字段映射</string>
    </property>
    <widget class="QTableWidget" name="pointTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>591</width>
       <height>281</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>250</number>
     </attribute>
     <column>
      <property name="text">
       <string>目标表关联字段名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>导入表关联字段名</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <zorder>page1</zorder>
  <zorder>page2</zorder>
  <zorder>page3</zorder>
  <zorder>prePushButton</zorder>
  <zorder>nextPushButton</zorder>
  <zorder>okPushButton</zorder>
  <zorder>page0</zorder>
 </widget>
 <resources>
  <include location="../PipeToolRes.qrc"/>
 </resources>
 <connections/>
</ui>
